python/plugins/web-app-test-server/README.md

```diff
@@ -0,0 +1,32 @@
+# sd-web-app-test-server
```

Comment:
update name to remove sd prefix (not needed in recipes)

Reply:
Removed prefix.